source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,263 @@
+
+### Footer
+
+`<footer role="contentinfo">` should be used for the footer of the site.
+
+## Individual element guidance
+
+### Headings
+
+Headings should be in order - for example `<h1>` then `<h2>`, but not `<h1>` then `<h3>`.
+
+There should only be one `<h1>` element in a page.
+
+### Text emphasis
+
+`<b>` and `<i>` tags make text bold and italic but have no semantic meaning; it
+is purely for visual styling.
+
+`<strong>` and `<em>` have the same effect visually, but have semantic meaning.
+In other words, the words they encapsulate are more important than the surrounding
+words.
+
+An example usage of `<strong>` is if we want to show the user their current location
+within a navigational list.
+
+```html
+<nav role="navigation">
+  <ol>
+    <li><a href="/">Home</a></li>
+    <li><a href="/business">Business</a></li>
+    <li><strong>Business Rates</strong></li>
+  </ol>
+</nav>
+```
+
+### Images
+
+Small, fixed size images can be implemented using the `img[src]` tag. Larger images should
+come in different sizes optimised for different devices, and [implemented responsively
+using `srcset`](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images).
+
+Modern formats such as [webp](https://developers.google.com/speed/webp/) offer the best
+compression methods (better for our users), but JPG/PNG fallbacks should always be provided.
+
+```html
+<picture>
+  <source srcset="example-320w.webp 320w,
+                  example-480w.webp 480w,
+                  example-800w.webp 800w"
+    sizes="(max-width: 320px) 280px,
+           (max-width: 480px) 440px,
+           800px"
+    type="image/webp" />
+  <!-- Fallback for browsers that support srcset but not webp -->
+  <source srcset="example-320w.jpg 320w,
+                  example-480w.jpg 480w,
+                  example-800w.jpg 800w"
+    sizes="(max-width: 320px) 280px,
+           (max-width: 480px) 440px,
+           800px"
+    type="image/jpeg" />
+  <!-- Fallback for browsers that do not support srcset -->
+  <img src="fallback.jpg" alt="This is my alt text" />
+</picture>
+```
+
+The `alt` tag should be used to describe the image to visually impaired users. If we were doing
+a presentation with a slide of the image but had a blind person in the room, how would you make
+sure that person wasn't missing any of the information?
+
+It is perfectly acceptable to have an empty `alt=""` attribute if the image is a background or
+presentational image. This explicitly tells the screen reader that there is no alternative text
+and the image gets skipped.
+
+Note that an empty `alt=""` is _not_ the same as having no alt attribute. If the attribute is
+missing, screen readers will read out the image filename instead, which is often not very useful.
+
+### Buttons
+
+Buttons should be used for client-side interactivity that doesn't change the destination of the
+page (e.g. a close button on a modal).
```

Comment:
If we're going to talk about buttons I'd do so in the wider context of guidance on how to build a form (I'm not sure this is covered yet?).